docs/rtc/index.md

```diff
@@ -1,8 +1,17 @@
 # Real Time Collaboration
 
-JupyterLite enables basic support for Real Time Collaboration over WebRTC.
+JupyterLite can be extended to enable basic support for Real Time Collaboration over
+WebRTC.
+
+```{note}
+Prior to version `0.1.0b5`, the WebRTC document provider was built directly into
```

Comment:
This will be the case in the next `0.1.0b6` release:

```suggestion
Prior to version `0.1.0b6`, the WebRTC document provider was built directly into
```